analysis/extreme_events_study/README.md

```diff
@@ -0,0 +1,39 @@
+# Extreme Events Study
+
+## How to get dataframes from the output of `extreme_events.py`?
+
+There are multiple dataframes stored as the values of the `dataframe_accumulator` objects in the `dict` output.
+
+```
+import pickle
+import gzip
+
+# Load output
+with gzip.open("path/to/output/file", "rb") as infile:
+    output = pickle.load(infile)
+
+# Get dataframes by keys (e.g. nleps, pt_j)
+df_nleps = output["nleps"].value
+df_pt_j = output["pt_j"].value
+```
+
+## How to get the yields from the MC samples?
+
+### 1. Use the output histogram from `topeft.py`
+
+* When initializing the histogram, adjust the bins to include the top events with clear divisions. If the bin is not there, add the event quantity to the variable list. Then add the bin to the histogram.
+* Run `topeft.py` on the MC samples for a specific group of events. An example of top jet multiplicity events in signal samples:
+
+```
+python work_queue_run.py ../../topcoffea/cfg/mc_signal_samples_NDSkim.cfg --hist-list njets --skip-cr --do-np
+```
+
+* Run `get_histo_yield.py` to get the yields. 
+
+### 2. Use the output dataframe from `extreme_events.py`
```

Comment:
Is this section of the description still related to getting output from `topeft`? It seems to be a sub section of the `How to get the yields from the MC samples` section, but it seems to be about using the output data frame from the extreme events processor.. maybe this subsection could be reorganized to go with the "How to get dataframes from the output of `extreme_events.py`" section?

Reply:
I was thinking about making two sections. The first one is getting the actual data in the form of dataframes from the extreme events processor. The second one is getting the yields from the MC samples. There were two ways of getting the yields: from `topeft` and from `extreme_events`. Since it is less likely to make a mistake with `topeft`, I put that as the first subsection. Would it be clearer if I add this explanation at the very beginning before the two sections?

Reply:
Yes, I think that would make it more clear